api/routes/api_teams.py

```diff
@@ -25,7 +24,8 @@ def get_teams_sport_endpoint():
         data = request.get_json()
         dto = TeamsLeagueDTO().load(data)
         pagination = Pagination(**data)
-        league_teams = get_teams(dto, pagination)
+        data_manager = TeamsDataManager(dto)
+        league_teams = data_manager.get_teams_data(pagination)
```

Comment:
MINOR:

it's better to separate "return" statement with a new line to make it stand out. Improves readability